### PR TITLE
Cleaned up comments in views directory and removed old unused code

### PR DESCRIPTION
**Pull request by jordancolbeth**

### CODE REVIEW

File: views/dashboard.handlebars [hunk 1]
```diff
@@ -25,7 +25,6 @@
                             <label for="sort-order">Sort Ads by: </label>
                         </div>
                         <div id="sortingContainer">
-                            <!-- the stuff above was just me playing around with function and buttons, can be disregarded or deleted -->
                             <!-- below is the actual form for sorting ads based on the filter, currently there's nothing that shows which filter they are set to, and I might have flipped
```
Comment (rorrell) [inline views/dashboard.handlebars]: This comment can also be removed; that's from Gabby but she fixed the code, so the comment is no longer relevant (I was the reviewer so I remember and I am in part to blame for its still being there 🤭 ).

**Review by rorrell — CHANGES_REQUESTED** [review]
Just one small thing

File: views/dashboard.handlebars (hunk 1 above)
Comment (jordancolbeth) [reply views/dashboard.handlebars]: Ok thanks!

**Review by rorrell — APPROVED** [review]
✔️

**Pull request merged**